packages/web3-react/README.md

```diff
@@ -0,0 +1,129 @@
+# Web3 react helpers
+
+Web3 react helpers for Lido Finance projects.
+Part of [Lido JS SDK](https://github.com/lidofinance/lido-js-sdk/#readme)
+
+## Install
+
+```bash
+yarn add @lido-sdk/web3-react
+```
+
+## Provider
+
+To use hooks from `@lido-sdk/web3-react` you need to wrap your app with `ProviderWeb3`. ProviderWeb3 also contains `ProviderSDK` from `@lido-sdk/react`. No need to wrap again.
+
+```tsx
+import { CHAINS } from '@lido-sdk/constants';
```

Comment:
How to use it with non Ethereum based network? As far as I remember `@lido-sdk/constants` has very limited amount of CHAINS

Reply:
It is not suited for non Ethereum based networks.
Later, the web3-react package will be deprecated at all, so we will be able to support all networks.